sycl/source/detail/config.cpp

```diff
@@ -161,8 +161,9 @@ void dumpConfig() {
 #undef CONFIG
 }
 
-// Array is used by SYCL_DEVICE_FILTER and SYCL_DEVICE_ALLOWLIST and
-// ONEAPI_DEVICE_SELECTOR
+// Array is used by SYCL_DEVICE_ALLOWLIST and ONEAPI_DEVICE_SELECTOR.
+// TODO: host device type will be removed once sycl_ext_oneapi_filter_selector
+// is removed.
```

Comment:
Do you know if we should remove it under the same task (even though in a separate PR)?

Reply:
AFAIK, from https://github.com/intel/llvm/pull/12007#discussion_r1407724969, we should perhaps first depreciate sycl_ext_oneapi_filter_selector and then, eventually remove or replace it.  I think it would be better to do that in a separate task as it would require more discussion and it is loosely related to the current task of removing depreciated variables.